source/patterns/@aws-solutions-constructs/core/lib/vpc-defaults.ts

```diff
@@ -42,12 +42,17 @@ export function DefaultIsolatedVpcProps(): ec2.VpcProps {
  */
 export function DefaultPrivateVpcProps(): ec2.VpcProps {
   return {
-    natGateways: 0,
+    natGateways: 1,
```

Comment:
Hold on, this is a HUGE change. NAT Gateways cost a fortune and you just added them to every construct. You just changed this into PublicPrivate VPC, a configuration we already handle elsewhere.

Reply:
It's also not well architected (I'm pretty sure). If you have 2 az, then you should have 2 NAT Gateways for availability.

Reply:
I will change it to the original configuration but luckily this function was only used in the lambda-elasticsearch construct